### DISCUSSION TITLE
how can i add the latest commit to my maven project?

### DISCUSSION THREAD

**Discussion by vinikjkkj** [Auties00/WhatsappWeb4j#273] [General]
I'm trying to use this api, but I can't because the maven repository version has the bug:

Unknown privacy option: calladd

and i am a new user of java and maven, i wanna add the last commit to my maven project, sorry for the silly question

**Comment by Auties00** [comment]
clone the repo and run
mvn clean install

**Reply by vinikjkkj** [reply]
thx
